test/scripts/run-gateway-conformance.sh

```diff
@@ -58,5 +58,7 @@ else
   git clone https://github.com/kubernetes-sigs/gateway-api
   cd gateway-api
   git checkout "${GATEWAY_API_VERSION}"
-  go test ./conformance -gateway-class=contour -supported-features=HTTPRouteQueryParamMatching,HTTPRouteMethodMatching
+  # Keep the list of supported features in sync with
+  # test/conformance/gatewayapi/gateway_conformance_test.go.
+  go test ./conformance -gateway-class=contour -supported-features=ReferenceGrant,TLSRoute,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching
```

Comment:
I think we'll actually be able to get rid of this code branch now that we're not running `main` Gateway API conformance against Contour's `main` branch, but keeping it around for now just to ensure we don't need it for something else.

Reply:
yeah since we're using a static commit now, its a little less important, but nice to have for local dev tbh, esp. when using Contour to test channges to the conformance tests